### ISSUE TITLE
[BUG] Lifting 2 fingers while resizing specific windows introduces noticeable lag

### ISSUE THREAD

**Issue by RndUsr123** [austinnixholm/ThreeFingerDrag#13]
As title says, lifting 2 fingers while resizing windows might drastically reduce resizing performance
A few examples:

- empty explorer folder: no delay
- explorer folder containing 10~20 mixed files and folders (SSD): slight to moderate delay
- explorer folder containing 50 empty .txt files (SSD): very obvious delay
- explorer folder containing >150 mixed files and folders, totaling >300GB (high-end SSD): very obvious delay
- explorer folder containing 6 folders totaling >350GB (HDD): no delay

Notes: 
- This is mostly consistent regardless of folder view settings
- Windows were continuously resized in a circular pattern

As this doesn't happen with 3 fingers or a mouse, it's probably not an OS or hardware limitation.

**Comment by austinnixholm** [comment]
Took some time to find a solution, and I found that the reason this lag occurs is due to an overlap of movements from both ThreeFingerDrag and the default touchpad driver.

To fix this I've implemented a small customizable delay that will occur when switching to one finger mid-gesture, which will allow the default driver movement to take precedence. 

This seems to be a solid workaround on my Magic Trackpad 2, so let me know how it works for you! 
 [Version 1.2.6.3-SNAPSHOT](https://github.com/austinnixholm/ThreeFingerDrag/releases/tag/v1.2.6.3-SNAPSHOT) also features official support for portable mode.
